src/bridge.cpp

```diff
@@ -125,7 +125,7 @@ create_bidirectional_bridge(
   const std::string & topic_name,
   size_t queue_size)
 {
-  RCLCPP_INFO(ros2_node->get_logger(), "create bidirectional bridge for topic " + topic_name);
+  RCLCPP_INFO(ros2_node->get_logger(), ("create bidirectional bridge for topic " + topic_name).c_str());
```

Comment:
Clang still may not like this, as you're passing in a nonliteral.

How about the following?
```
RCLCPP_INFO(ros2_node->get_logger(), "create bidirectional bridge for topic %s", topic_name.c_str());
```
or
```
RCLCPP_INFO(ros2_node->get_logger(), "%s", ("create bidirectional bridge for topic " + topic_name).c_str());
```

Reply:
I went with:
```
  RCLCPP_INFO(ros2_node->get_logger(), "create bidirectional bridge for topic %s", topic_name.c_str());
```